<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="stock_A_List_label">
       <property name="text">
        <string>Stock_A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="stock_A_tableWidget">
       <row>
        <property name="text">
         <string>卖五</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>卖四</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>卖三</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>卖二</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>卖一</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买一</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买二</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买三</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买四</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买五</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>价格</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>量</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="stock_B_List_label">
       <property name="text">
        <string>Stock_B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="stock_B_tableWidget">
       <row>
        <property name="text">
         <string>卖五</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>卖四</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>卖三</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>卖二</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>卖一</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买一</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买二</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买三</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买四</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>买五</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>价格</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>量</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>成交明细</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>股票对</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="pairKey_label">
            <property name="text">
             <string>000001-600001</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="stock_A_label">
            <property name="text">
             <string>000001</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="stock_A_Direction_label">
            <property name="text">
             <string>Buy</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>成交价</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="stock_A_Price_doubleSpinBox"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>成交量</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>成交时间</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateTimeEdit" name="dealTime_dateTimeEdit"/>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="stock_A_Vol_spinBox"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="stock_B_label">
            <property name="text">
             <string>600001</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="stock_B_Direction_label">
            <property name="text">
             <string>Sell</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>成交价</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="stock_B_Price_doubleSpinBox"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>成交量</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="stock_B_Vol_spinBox"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>配对系数</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="pair_Value_label">
            <property name="text">
             <string>1.2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>交易类型</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="trade_Type_label">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>85</height>
        </size>
       </property>
       <property name="title">
        <string>阈值</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="open_label">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="close_label">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Stocp</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="stop_label">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
